src/components/SearchBar.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import { Input } from '@material-ui/core'
+
+const inputStyle = {
+  color: '#000',
+  border: '1px solid #000',
+  width: '100%',
+  boxShadow: 'none'
+}
+const SearchBar = ({ sortFunc }) => {
+  const [value, setValue] = useState('')
+  const handleSubmit = e => {
+    e.preventDefault()
```

Comment:
What is the purpose of preventing default here?

Reply:
deleted